egs2/librimix/enh_diar1/local/prepare_kaldifiles.py

```diff
@@ -0,0 +1,357 @@
+import os
```

Comment:
Does this file cover all the purposes of `egs2/librimix/diar1/local/prepare_diarization.py`? If so, maybe two files can be merged?

Reply:
There will be some extra files (spk & noise scp files) if we use this file for diarization. If that is OK, we can merge them.

Reply:
I don't know, maybe some flags to decide whether to generate the extra files? Or in `data.sh`, remove these files if not necessary.